### DISCUSSION TITLE
Easter Island is a part of which South American country?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#259] [Galaxy Trivia] [answered]
**Hint:** 🌶️

**Comment by muink** [comment, marked as answer]
Chile

**Reply by ParanoidUser** [reply]
Bingo! Congratulations on your well deserved bronze 🥉[Galaxy Brain](https://github.com/live627?tab=achievements&achievement=galaxy-brain) badge. Continue to unlock the next badge levels (Silver for 16 answers), and don't hesitate to post your own questions to the community. 😊🎉

**Comment by ItsNotAlexy** [comment]
Chile

**Comment by ghost** [comment]
Chile
